api/soap/mc_issue_api.php

```diff
@@ -1597,3 +1597,59 @@ function mci_issue_data_as_header_array( $p_issue_data ) {
 
 		return $t_issue;
 }
+
+/**
+ * Set Issue Handler
+ *
+ * @param string $p_username The name of the user trying to add the issue.
+ * @param string $p_password The password of the user.
+ * @param integer $p_issue_id 
+ * @param integer $p_handler_id 
+ * @return boolean
+ */
+function mc_issue_set_handler( $p_username, $p_password, $p_issue_id, $p_handler_id ) {
+	global $g_project_override;
+
+	$t_user_id = mci_check_login( $p_username, $p_password );
+	if( $t_user_id === false ) {
+		return mci_soap_fault_login_failed();
+	}
+
+	if( !bug_exists( $p_issue_id ) ) {
+		return SoapObjectsFactory::newSoapFault('Client', "Issue '$p_issue_id' does not exist.");
+	}
+
+	if( bug_is_readonly( $p_issue_id ) ) {
+		return SoapObjectsFactory::newSoapFault('Client', "Issue '$p_issue_id' is readonly");
+	}
+
+	$t_project_id = bug_get_field( $p_issue_id, 'project_id' );
+
+	if( !mci_has_readwrite_access( $t_user_id, $t_project_id ) ) {
+		return mci_soap_fault_access_denied( $t_user_id );
+	}
+
+	$g_project_override = $t_project_id; // ensure that helper_get_current_project() calls resolve to this project id
+
+    if( is_null($p_handler_id) && $p_handler_id <= 0) {
```

Comment:
Did you mean to have an || (OR) here?   Adjust spacing around parenthesis to match rest of code and indentation.